### PR TITLE
[ADF-3810] Should rearrange toolbar actions in Viewer

### PR DESCRIPTION
**Pull request by gmandakini**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/pages/adf/viewerPage.ts [hunk 1]
```diff
@@ -185,6 +186,16 @@ export class ViewerPage {
         Util.waitUntilElementIsVisible(this.closeButton);
     }
 
+    checkButtonOrderInViewerToolbar() {
+        this.checkToolbarIsDisplayed();
+        expect(this.buttonList.count()).toBe(5);
+        expect(this.buttonList.get(0).getAttribute('title')).toEqual(this.downloadButton.getAttribute('title'));
+        expect(this.buttonList.get(1).getAttribute('title')).toEqual(this.printButton.getAttribute('title'));
+        expect(this.buttonList.get(2).getAttribute('title')).toEqual(this.fullScreenButton.getAttribute('title'));
```
[the remark this answers is not in the document]
Comment (gmandakini) [reply e2e/pages/adf/viewerPage.ts]: the changes are done. I have updated the button list to return only the last button and assering that in the test that the last button displayed is the 'More Actions' menu button.

File: e2e/pages/adf/viewerPage.ts [hunk 2]
```diff
@@ -185,6 +186,16 @@ export class ViewerPage {
...
+        expect(this.buttonList.get(3).getAttribute('title')).toEqual(this.infoButton.getAttribute('title'));
+        expect(this.buttonList.get(4).getAttribute('title')).toEqual(this.moreActionsMenu.getAttribute('title'));
+    }
```
[the remark this answers is not in the document]
Comment (gmandakini) [reply e2e/pages/adf/viewerPage.ts]: deleted the method in the viewerpage and added the expect statement in the test itself.

File: e2e/pages/adf/viewerPage.ts [hunk 3]
```diff
@@ -62,12 +62,16 @@ export class ViewerPage {
     activeTab = element(by.css('div[class*="mat-tab-label-active"]'));
     toolbarSwitch = element(by.id('adf-switch-toolbar'));
     toolbar = element(by.id('adf-viewer-toolbar'));
+    lastButton = element.all(by.css('#adf-viewer-toolbar mat-toolbar > button[data-automation-id*="adf-toolbar-"]')).last();
```
Comment (cristinaj) [inline e2e/pages/adf/viewerPage.ts]: It's not enough to use just: button[data-automation-id*="adf-toolbar-"] ?

File: e2e/pages/adf/viewerPage.ts [hunk 4]
```diff
@@ -185,6 +186,10 @@ export class ViewerPage {
         Util.waitUntilElementIsVisible(this.closeButton);
     }
 
+    getLastButtonDisplayed() {
```
Comment (cristinaj) [inline e2e/pages/adf/viewerPage.ts]: I would rename this method to getLastButtonTitle() and return in here the value of the 'title' attribute and just use it in the test. Same for moreActionsMenu.

File: e2e/pages/adf/viewerPage.ts (hunk 3 above)
Comment (gmandakini) [reply e2e/pages/adf/viewerPage.ts]: This is the best locator I could find to get the list of buttons in the Viewer toolbar. But open to suggestions.

File: e2e/pages/adf/viewerPage.ts (hunk 4 above)
Comment (gmandakini) [reply e2e/pages/adf/viewerPage.ts]: done, thanks

File: e2e/core/viewer/viewer-content-services-component.e2e.ts [hunk 5]
```diff
@@ -271,6 +271,17 @@ describe('Content Services Viewer', () => {
         viewerPage.clickCloseButton();
     });
 
+    it('[C279922] Should display the buttons in order in the adf viewer toolbar', () => {
```
Comment (marouanbentaleb) [inline e2e/core/viewer/viewer-content-services-component.e2e.ts]: Could you change this ID ?

Comment (gmandakini) [reply e2e/core/viewer/viewer-content-services-component.e2e.ts]: done

**Pull request merged**